viewport/viewport.go

```diff
@@ -331,18 +331,15 @@ func (m Model) updateAsModel(msg tea.Msg) (Model, tea.Cmd) {
 			}
 		}
 
-	case tea.MouseMsg:
-		if !m.MouseWheelEnabled || msg.Action != tea.MouseActionPress {
-			break
-		}
+	case tea.MouseWheelMsg:
 		switch msg.Button {
```

Comment:
With https://github.com/charmbracelet/bubbletea/pull/1111, this is going to be `msg.Button()`